src/osx.md

```diff
@@ -22,7 +22,7 @@ installed in `~/opt`. This is so we don't clobber any version of them that you
 may have already installed. The source code for these tools are downloaded in
 `~/src`.
 
-[this script]: https://gist.githubusercontent.com/emkay/a1214c753e8c975d95b4/raw/cc73290537898799fe1168edfbc5a41351184670/build-grub-osx.sh
+[this script]: https://intermezzos.github.io/book/appendix/osx-install.html
```

Comment:
ah, last nit: can you make this link `/appendix/osx-install.html`? the relative link means it will work locally as well as when it's published